embedded-couchbase/src/main/java/com/playtika/test/couchbase/rest/StartupUtils.java

```diff
@@ -0,0 +1,14 @@
+package com.playtika.test.couchbase.rest;
+
+import com.playtika.test.couchbase.CouchbaseProperties;
+import lombok.experimental.UtilityClass;
+
+import static java.lang.String.format;
+
+@UtilityClass
+public class StartupUtils {
+
+    public static String credentials(CouchbaseProperties couchbaseProperties) {
```

Comment:
can you create CouchbaseProperties.credentials?
and also you can keep validation there